crates/ra_assists/src/handlers/fill_match_arms.rs

```diff
@@ -114,6 +192,183 @@ mod tests {
 
     use super::fill_match_arms;
 
+    #[test]
+    fn partial_fill_multi() {
+        check_assist(
+            fill_match_arms,
+            r#"
+            enum A {
+                As,
+                Bs(i32, Option<i32>)
+            }
+            fn main() {
+                match A::As<|> {
+                    A::Bs(_, Some(_)) => (),
+                }
+            }
+            "#,
+            r#"
+            enum A {
+                As,
+                Bs(i32, Option<i32>)
+            }
+            fn main() {
+                match <|>A::As {
+                    A::Bs(_, Some(_)) => (),
+                    A::As => (),
+                    _ => (),
```

Comment:
I think it's fine to not add `_`. The goal is not to produce the code that compiles, the goal is to help the user, and, if we are not sure what pattern do they have in mind, it's better to do nothing.

Reply:
I do think a nice side effect of adding the `_` is that the assist gives you something that compiles right away.

Reply:
Not tracking the `_` makes this logic much simpler.

Reply:
My gut feeling is that `_` would be just something that users would have to delete and rewrite. I'd start with simpler approach without `_` and add it back later, if we deem it later. We [do](https://github.com/rust-analyzer/rust-analyzer/pull/477#issuecomment-574225723) adjust things afterwards.